src/planner/Maintain.h

```diff
@@ -573,47 +573,62 @@ class DropEdgeIndex final : public DropIndexNode {
         : DropIndexNode(qctx, Kind::kDropEdgeIndex, input, std::move(indexName), ifExists) {}
 };
 
-class RebuildIndexNode : public SingleInputNode {
-protected:
-    RebuildIndexNode(QueryContext* qctx,
-                     Kind kind,
-                     PlanNode* input,
-                     std::string indexName)
-        : SingleInputNode(qctx, kind, input)
-        , indexName_(std::move(indexName)) {}
-
+class ShowCreateTagIndex final : public DescIndexNode {
 public:
-    const std::string& getIndexName() const {
-        return indexName_;
+    static ShowCreateTagIndex* make(QueryContext* qctx,
+                                    PlanNode* input,
+                                    std::string indexName) {
+        return qctx->objPool()->add(new ShowCreateTagIndex(qctx->genId(),
+                                    input,
+                                    std::move(indexName)));
     }
 
-    std::unique_ptr<cpp2::PlanNodeDescription> explain() const override;
+private:
+    ShowCreateTagIndex(int64_t id,
```

Comment:
Now the plan node construction parameter has been changed, you should pass the QueryContext pointer in.

Reply:
Thanks. Has fixed.